skimage/filters/_gaussian.py

```diff
@@ -19,8 +19,8 @@ def _convert_input(image, preserve_range):
 
 
 def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
-             multichannel=None, preserve_range=False):
-    """Multi-dimensional Gaussian filter
+             multichannel=None, preserve_range=False, truncate=4.0):
```

Comment:
Yes, but this would require to manage a deprecation for users who do not specify parameters with keywords (a bad thing... I admit).